src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Vipps/VippsPaymentsResource.cs

```diff
@@ -0,0 +1,37 @@
+﻿using SwedbankPay.Sdk.Common;
+using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Vipps
+{
+    public class VippsPaymentsResource : ResourceBase, IVippsResource
+    {
+        public VippsPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public async Task<IVippsPaymentReponse> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
+        {
+            if (id == null)
+            {
+                throw new ArgumentNullException(nameof(id));
+            }
+
+            Uri url = id.GetUrlWithQueryString(paymentExpand);
+
+            var paymentResponseContainer = await HttpClient.GetAsJsonAsync<VippsPaymentResponseDto>(url);
+            return new VippsPaymentResponse(paymentResponseContainer, HttpClient);
+        }
+
+        public async Task<IVippsPaymentReponse> Create(VippsPaymentRequest paymentRequest,
+                                                            PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)